src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+
+    public static void main(String[] args) throws IOException {
+        System.out.println("    Hello from\n" + LOGO);
+        System.out.println(LINE_SEPERATOR +
+                "\n    Hello! I'm Duke\n" +
+                "    What can I do for you?\n" +
+                LINE_SEPERATOR);
+        Scanner sc = new Scanner(System.in);
+        String sourceFilePath = "data" + File.separator + "task.txt";
+        File source = new File(sourceFilePath);
+        //FileHandler fileHandler;
+        //FileTaskWriter fileTaskWriter;
+        FileTaskWriter fileTaskWriter = new FileTaskWriter(sourceFilePath);
+        try {
+            if(!source.exists()) {
+                if (!source.getParentFile().exists()) {
+                    source.getParentFile().mkdirs();
+                    source.createNewFile();
+                }
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+            System.out.println(LINE_SEPERATOR + "    Duke can't find the task list.\n" + LINE_SEPERATOR);
+        }
+        FileHandler fileHandler = new FileHandler(source);
+        TaskList tasks = new TaskList(fileHandler.parseToArraylist());
+        //FileWriter sourceToWrite = new FileWriter(sourceFilePath);
+        //fileTaskWriter = new FileTaskWriter(sourceFilePath);
+        String userInput = sc.nextLine();
+        while (!userInput.equals("bye")) {
+            try {
+                if (userInput.equals("list")) {
+                    tasks.printTaskList();
+                    userInput = sc.nextLine();
+                    continue;
+                }
+                String[] inputParts = userInput.trim().split("\\s+",2);
+                switch (inputParts[0]){
+                case "done":
+                    if(inputParts.length == 1) {
+                        throw new DoneFormatException();
+                    }
+                    tasks.setTaskDone(inputParts[1]);
+                    break;
+                case "delete":
+                    if(inputParts.length == 1) {
+                        throw new DeleteFormatException();
+                    }
+                    tasks.deleteTask(inputParts[1]);
+                    break;
+                case "todo":
+                    if(inputParts.length < 2) {
+                        throw new TodoFormatException();
+                    }
+                    ToDo newTodo = new ToDo(inputParts[1],false);
+                    tasks.addTask(newTodo);
+                    break;
+                case "deadline":
+                    if (inputParts.length < 2) {
+                        throw new DeadlineFormatException();
+                    }
+                    String[] deadlineNameDoby = inputParts[1].split("/by");
+                    if(deadlineNameDoby.length < 2) {
+                        throw new DeadlineFormatException();
+                    }
+                    String deadlineName = deadlineNameDoby[0].trim();
```

Comment:
Remember to use camelCase for your variable names (in this case, deadlineNameDoBy)